src/demos/namespaced_example /app.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
```

Comment:
I think you have a space in the folder name? `namespaced_example /app.js`
let's go for `cds-videos-namespaced` for consistency